cmd/ddev/cmd/delete-images.go

```diff
@@ -1,26 +1,32 @@
 package cmd
 
 import (
+	"os"
+	"sort"
+	"strings"
+
 	"github.com/drud/ddev/pkg/dockerutil"
 	"github.com/drud/ddev/pkg/nodeps"
 	"github.com/drud/ddev/pkg/util"
 	"github.com/drud/ddev/pkg/version"
 	docker "github.com/fsouza/go-dockerclient"
 	"github.com/spf13/cobra"
-	"os"
-	"sort"
-	"strings"
 )
 
 // noConfirm: If true, --yes, we won't stop and prompt before each deletion
 var deleteImagesNocConfirm bool
 
+// deleteAllImages: If set, deletes all images created by ddev
+var deleteAllImages bool
```

Comment:
In general, I've been trying to use the simpler techniques instead of the variable techniques, although this pattern is widely seen. But if you use the pattern like https://github.com/drud/ddev/blob/a834bc76a17f83504bcad559f468ccc933693f1f/cmd/ddev/cmd/config.go#L560-L565 and https://github.com/drud/ddev/blob/a834bc76a17f83504bcad559f468ccc933693f1f/cmd/ddev/cmd/config.go#L305 then no variable is needed. You don't have to change this if you like it as is.

Reply:
I put the variable here to be consistent with the existing code.

Reply:
Right, and nothing wrong with it. I only started using the other technique a year or so ago. Just wanted to point out the other way.